bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/model/Evaluation.kt

```diff
@@ -0,0 +1,15 @@
+package io.bucketeer.sdk.android.internal.model
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class Evaluation(
```

Comment:
note: `io.bucketeer.sdk.android.BKTEvaluation` will be added as a separate class.